### ISSUE TITLE
no keyboard in responsive mode

### ISSUE THREAD

**Issue by Centaur** [christianalfoni/webpack-bin#100]
Reduce the browser width until the right preview pane disappears, you will not be able to input using keyboard in the code area.

**Comment by christianalfoni** [comment]
Yeah, its because its the mobile version. Have no keyboard on mobile :-)

**Comment by christianalfoni** [comment]
You expect to behave differently?

**Comment by Centaur** [comment]
When developing on desktop, I split the whole screen and give the browser only the left part. And it won't hurt if add the keyboard support, I think.

**Comment by christianalfoni** [comment]
Aha, I see, hm... could I ask how many pixels wide that gets then? Its just that mobile version currently has no support for editing the BIN. Its not that much effort implementing it, just exploring other options. Maybe actually check if it is a mobile phone. If you could provide a screenshot that would also help getting some ideas :) Thanks!

**Comment by Centaur** [comment]
This is my current screen. I think it's better to check if it is a mobile phone so that we don't need to know the actual screen width.
![2016-03-31 08 30 37](https://cloud.githubusercontent.com/assets/265524/14162023/c9c6d0b4-f71a-11e5-95de-10c8f774a41c.png)

**Comment by christianalfoni** [comment]
Yes, agreed, will do that implementation on next version. Sorry it is taking a bit time, but been occupied with some other stuff lately
